whisper.py

```diff
@@ -95,6 +95,9 @@ def _py_fallocate(fd, offset, len_):
 CACHE_HEADERS = False
 AUTOFLUSH = False
 FADVISE_RANDOM = False
+# Buffering setting applied to all operations that do *not* require
+# a full scan of the file in order to minimize cache thrashing.
+BUFFERING = 0
```

Comment:
The default is system default, which is 8k usually on linux. That's super wasteful for things such as update_many which only read the header and a few points.